### PR TITLE
refactoring of seismic reflection

### PR DESCRIPTION
**Pull request by thast**
-reorganisation of the different sections
-few typos

ongoing work:
-renew figures for better resolution
-correct typos maths
-add figures from presentations and labs

### CODE REVIEW

**Comment by thast** [comment]
I got some problems with git: it is not detecting a couple of new .rst I made... hopefully will be able to fix it soon.

File: content/seismic/reflection/seismic_reflection_common_middle_points.rst [hunk 1]
```diff
@@ -1,6 +1,6 @@
-.. _seismic_reflection_common_reflection_points:
+.. _seismic_reflection_common_middle_points:
 
-Common Reflection Points
+Common Midpoint Points
```
Comment (lheagy) [inline content/seismic/reflection/seismic_reflection_common_middle_points.rst]: maybe "Common Midpoint Gathers" (midpoint points is a bit repetitive). Thoughts @dougoldenburg?

**Comment by lheagy** [comment]
@thast : can you provide a list of what you consider to be todos before merging this? eg 
```
- [ ] TODO1
```
Thanks!

**Comment by thast** [comment]
@lheagy 👍 

I think it is in good enough shape to be merge now. The structure is what we agree on during the last seismic meeting.

The next TODO would be:
- having @dougoldenburg reviewing the updates.

What was done:
-reorganisation of differents section
-updates of pixelated images, either by finding the originals or by finding new examples 
-smalls fixs, texts or equations updates

**Comment by lheagy** [comment]
nice @thast! And for the images, we have copyright permission to use all of them?

**Comment by lheagy** [comment]
If so, feel free to merge :)

**Comment by thast** [comment]
Images come from subsurfwiki or wikipedia or USGS. They were tagged as Common Creative Licenses and the references to the originals and authors have been added as legends.

**Pull request merged**